src/core/content/main.ts

```diff
@@ -25,15 +27,23 @@ async function main() {
  */
 async function fetchConnector(): Promise<void> {
 	const connector = await getConnectorByUrl(window.location.href);
+	console.log('bbbbb', connector, window.location.href);
 	if (!connector) {
 		return;
 	}
 
+	// Don't run the connector in frames if it's not allowed to run in frames
+	if (window !== top && !connector.allFrames) {
```

Comment:
What does the first conditional do? Not familiar with this :)

Reply:
window === top if content script is not running in a frame/iframe